CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Release History
 
+## 1.16.0 (2024-05-07)
+
+### New Features
+
+- Added snowflake.snowpark.Session.lineage.trace to explore data lineage of snowfake objects.
+- Support stored procedure registration with packages given as Python modules.
+- Added support for structured type schema parsing.
+
+### Bug Fixes
+
+- Fixed a bug that when inferring a schema, single quotes are added to stage files that already have single quotes.
+
+### Local Testing Updates
+
+#### New Features
+
+- Added support for StringType, TimestampType and VariantType data conversion in the mocked function `to_date`.
+- Added support for the following APIs:
+  - snowflake.snowpark.functions
+    - get
+    - concat
+    - concat_ws
+
+#### Bug Fixes
+
+- Fixed a bug that caused NaT and NaN values to not be recognized.
+- Fixed a bug when inferring schema, single quotes are added to stage files already have single quotes.
+- Fixed a bug in DataFrameReader.csv unable to handle quoted values containing delimiter.
```

Comment:
- Fixed a bug in DataFrameReader.csv unable to handle quoted values containing delimiter.
->
- Fixed a bug where DataFrameReader.csv was unable to handle quoted values containing a delimiter.